content/docs/reference/spec/migration/platform-api.md

```diff
@@ -0,0 +1,42 @@
++++
+title="Platform API 0.3 -> 0.4"
++++
+
+<!--more-->
+
+This guide describes changes from platform API 0.3 to 0.4, who these changes are relevant for,
+as well as any actions that may be necessary.
+
+This guide is most relevant to platform operators.
+
+### Windows support
+
+Windows image builds are now supported! See the [Windows build guide](/docs/app-developer-guide/build-a-windows-app)
+for more details.
+
+### Multicall launcher
+
+When running the exported application image, it is now possible to provide arguments to pre-defined process types. 
+The new flow is described [here](/docs/app-developer-guide/run-an-app/#run-a-multi-process-app).
+As part of these changes, the exporter will not set, and the launcher no longer accepts, the `CNB_PROCESS_TYPE` variable.
+See the associated [RFC](https://github.com/buildpacks/rfcs/blob/main/text/0045-launcher-arguments.md) and [spec PR](https://github.com/buildpacks/spec/pull/118)
```

Comment:
Personally I think that adding links to the RFC and spec PR is too much for this type of document, but I think we should allow the reader to get more information about the new feature in a different way.
What do you think about adding another page or section in this page with more details about each change in the new API?